packages/browser-integration-tests/suites/public-api/captureMessage/parameterized_message/test.ts

```diff
@@ -0,0 +1,21 @@
+import { expect } from '@playwright/test';
+import type { Event } from '@sentry/types';
+
+import { sentryTest } from '../../../../utils/fixtures';
+import { getFirstSentryEnvelopeRequest, shouldSkipTracingTest } from '../../../../utils/helpers';
+import { shouldSkipReplayTest } from '../../../../utils/replayHelpers';
+
+sentryTest('should capture a parameterized representation of the message', async ({ getLocalTestPath, page }) => {
+  if (shouldSkipReplayTest() || !shouldSkipTracingTest()) {
```

Comment:
I think this is not gonna be enough. We need to skip all [`bundle_` variants](https://github.com/getsentry/sentry-javascript/blob/94bc3494f6f4d0c9b1f70ce8313f6dedfb090a68/.github/workflows/build.yml#L551-L565)  for now. These two functions won't do that. 

basically like in this test:
https://github.com/getsentry/sentry-javascript/blob/94bc3494f6f4d0c9b1f70ce8313f6dedfb090a68/packages/browser-integration-tests/suites/replay/replayShim/test.ts#L8-L13

Reply:
Thanks for the hint about skipping the test for all bundle builds.
I made changes, should work now.

Can you also explain how you are testing such functions for public API for CDN bundles? Because importing functions directly to tests will not work, and I'm wondering how I can test them. Just an example would be enough, I didn't find anything similar in `browser-integration-tests`. Will do it as a separate PR as you mentioned

Reply:
So as soon as we export `parameterize` from @sentry/browser, our CDN bundles will also include it as a top level function to call on the `Sentry` object. Meaning, CDN bundle users can call

```js
Sentry.captureMessage(Sentry.parameterize`This is a log statement with ${x} and ${y} params`);
```

we can adjust the integration tests to do that and stop importing it from @sentry/utils.

Reply:
that's generally how we test public API in the browser integration tests, across NPM package (`esm`, `cjs`) and CDN bundle variants.

Reply:
For a simple example, take a look at this test: https://github.com/getsentry/sentry-javascript/blob/23ef22b115c8868861896cc9003bd4bb6afb0690/dev-packages/browser-integration-tests/suites/public-api/setTag/with_primitives/subject.js

calling `Sentry.parameterize` should work just like calling `Sentry.setTag` once we export it from the browser package.

Reply:
Thanks, just tried also throws timeouts locally, but works with esm, cjs builds. Will do separate PR for that, and will mention you to run tests there, let's see if it will work on cloud setup. A lot of other tests also failing for me locally because of timeout for example for bundle_tracing_replay_es6_min, so I'm not quite sure if it works or not.

For now please merge it, just pulled latest changes from `develop` and I'll do separate PR for running tests for all of the builds. Thanks!